src/Protocol/DFRN.php

```diff
@@ -2931,9 +2939,18 @@ public static function autoRedir(App $a, $contact_nick)
 				DBA::escape($baseurl),
 				DBA::escape($nurl)
 			);
-			if ((! DBA::isResult($r)) || $r[0]['id'] == remote_user()) {
+			if ((! DBA::isResult($r))) {
 				return;
 			}
+			// test if redirect authentication already succeeded
+			// Note that "contact" in the sense used in $contact_nick and the sense in the $remote[]["cid"]
```

Comment:
Please clarify the reference to "$contact_nick" that is absent from the below code.

Reply:
It is an argument of the function.  With latest commit, the comment now states that.